plot/plot.app.js

```diff
@@ -1157,16 +1285,65 @@
+                                                            var url=data.points[0].data.y_col_hyperlink_pattern[idx]+qCharacter+"pcid="+contextUrlParams.cid+"&ppid="+contextUrlParams.pid;
+                                                            window.open(url,'_blank');
+                                                        }
+                                                    }
+                                                    if(data.hasOwnProperty("points") && data.points[0].data.hasOwnProperty("graphic_link_pattern")){
+                                                        var idx=data.points[0].data.labels.indexOf(data.points[0].label.toString());
+                                                        if( data.points[0].data.graphic_link_pattern[idx]!=false && data.points[0].data.graphic_link_pattern[idx]!=undefined){
+                                                            var url=data.points[0].data.graphic_link_pattern[idx];
+                                                            window.open(url,'_blank');
+                                                        }
+                                                    }
+                                                }),
+                                                element[0].on('plotly_legendclick', function(data){
+                                                    // var contextUrlParams=ConfigUtils.getContextHeaderParams();
+                                                    if(data.hasOwnProperty("data")){
+                                                            if(data.data[0].hasOwnProperty("legend_markdown_pattern") ){
+                                                                var idx=data.data[0].labels.indexOf(data.label.toString());
+                                                                if(data.data[0].legend_markdown_pattern[idx]!=false && data.data[0].legend_markdown_pattern[idx]!=undefined){
+                                                                    window.open(data.data[0].legend_markdown_pattern[0],'_blank');
+                                                                    return false;
+                                                                }
+                                                            }
+
+                                                            if(data.data[0].hasOwnProperty("data_legend_pattern")){
+
+                                                                if(data.data[0].data_legend_pattern[0]!=false && data.data[0].data_legend_pattern[0]!=undefined){
+                                                                    //var qCharacter = data.data[0].legend_markdown_pattern[idx].indexOf("?") !== -1 ? "&" : "?";
+                                                                    var url=data.data[0].legend_markdown_pattern[idx];//+qCharacter+"pcid="+contextUrlParams.cid+"&ppid="+contextUrlParams.pid;
```

Comment:
This property is not consistent with the conditional statement we are in.

Reply:
Removed these set of lines(They were specific case I had added previously removed it completely)